app/src/main/java/org/oppia/android/app/onboarding/OnboardingFragmentPresenter.kt

```diff
@@ -50,18 +45,11 @@ class OnboardingFragmentPresenter @Inject constructor(
   }
 
   private fun setUpViewPager() {
-    onboardingPagerAdapter =
+    val adapter =
       OnboardingPagerAdapter(fragment.requireContext(), getOnboardingSlideFinalViewModel())
-    binding.onboardingSlideViewPager.adapter = onboardingPagerAdapter
-    // TODO (#2194 ): Remove this once the implementation is updated to ViewPager2
-    val isRTL = TextUtilsCompat
```

Comment:
is it working correctly for RTL?

Reply:
I guess there's some problem with the back button - 


https://user-images.githubusercontent.com/54636525/104893546-91727d80-5999-11eb-8e11-b87c20abc892.mp4

Reply:
@FareesHussain mentioning you here to help with RTL stuff.

Reply:
video looks fine
@prayutsu have a look at the todos in the `OnboardingFragmentPresenter` and `OnboardingPagerAdapter` and revert them
and then check if RTL works fine

Reply:
I checked it on the `develop` branch, and it looks ditto same as in this video.
Is it correct?

Reply:
Yes